behaviortree_ros2/include/behaviortree_ros2/bt_action_node.hpp

```diff
@@ -206,6 +206,15 @@ class RosActionNode : public BT::ActionNodeBase
     return rclcpp::Clock(RCL_ROS_TIME).now();
   }
 
+  rclcpp::Clock::SharedPtr get_clock()
+  {
+    if(auto node = node_.lock())
+    {
+      return node->get_clock();
+    }
+    return std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+  }
```

Comment:
First and foremost, please change the name of he method to respect the style of the class.

Also, would it make sense to avoid creating a new clock every time?

```suggestion
  rclcpp::Clock::SharedPtr getClock()
  {
    if(auto node = node_.lock())
    {
      return node->get_clock();
    }
    static auto clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
    return clock;
  }
```

Reply:
Thanks for your reply. About the style: I suppose then also logger() should be renamed to getLogger() to for consistency, alternatively this could be called just clock(), but I prefer the former as it is more descriptive. I think it makes sense to avoid making a new clock, we just need to make sure that this is not also an issue for now() which has a similar default return.

Reply:
`clock()` sounds good to me